src/components/FormElements/PubKeyPicker.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
```

Comment:
I refactored this component in my current PR and moved the validation outside. Thanks for handling it here! I'll fix any conflicts in my PR.